src/SDKs/ContainerService/ContainerService.Tests/Microsoft.Azure.Management.ContainerService.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="2.0.0-preview" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.5" />
```

Comment:
Please remove these refs.
Please use [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Advisor/Advisor.Tests/Advisor.Tests.csproj) as a template for the test csproj